docs/faq.md

```diff
@@ -57,6 +57,27 @@ Combine `default` and `when: false`.
 See [advanced prompt formatting docs][advanced-prompt-formatting]. If you need more
 power, see [below][how-can-i-alter-the-context-before-rendering-the-project].
 
+## How to "lock" a computed value?
+
+When you want to ensure that a computed value is set or locked during project initialization,
+for example if you want to store a dynamically computed `copyright_year` but ensure that it doesn't
+change upon later copier template updates, you can combine the above `default` and `when:false`
+configuration while also explicitly saving the value into your `{{ _copier_conf.answers_file }}.jinja`.
```

Comment:
Perhaps more descriptive and with a link?

```suggestion
configuration while also explicitly saving the value into your [answers file][the-copier-answersyml-file].
```

Reply:
```suggestion
configuration while also explicitly dumping the value to yaml with the [answers_file](/configuring/#the-copier-answersyml-file).
```